source/testing_govuk_pay/index.html.md.erb

```diff
@@ -95,6 +95,7 @@ The [PSP transaction fee](/reporting/#psp-fees) depends on which country the tes
 |Successful payment |5105105105105100| Mastercard | Debit | US |
 |Successful payment |5200828282828210| Mastercard | Debit | US |
 |Successful payment |371449635398431| American Express | Credit | US |
+|Requires 3DS check |4000002500003155| Visa | Credit | France
```

Comment:
It's missing a `|` at the end of the line. Not sure if this is required for it to format correctly

Reply:
@stephencdaly Good spot! I think I've added that now.